lib/types/index.ts

```diff
@@ -364,12 +364,18 @@ export interface TSRdfProjectSearchStage {
 
 export type TSRdfSearchStage = TSRdfBgpSearchStage|TSRdfFilterSearchStage|TSRdfConstructSearchStage|TSRdfProjectSearchStage;
 
+export interface TSRdfPrefixes {
+  expandTerm(term: string): string | null;
```

Comment:
What is the use case for having `null` as a possible return type for `expandTerm()`?

I understand that `expandRequiredTerm()` takes care of handling `null` but, given that `prefixes.expandTerm()` is only used within `expandRequiredTerm()`, wouldn't it be better to simply force `prefixes.expandTerm()` to always return either the original value (if unable to expand) or the expanded value (if present)? I think this is what the default functions https://github.com/beautifulinteractions/node-quadstore/blob/7f2395927cc0a3f8df9d5959780c509dd29bf631/lib/rdfstore.ts#L66-L69 are doing.

Reply:
Originally this was because the jsonld-context-parser had this signature, which in turn was because it's possible to disable a term entirely in a JSON-LD context. However, we've moved away from that signature anyway, so I agree to just drop the `null`.